mavros_extras/src/plugins/esc_telemetry.cpp

```diff
@@ -0,0 +1,122 @@
+#include <mavros/mavros_plugin.h>
+#include <mavros_msgs/ESCTelemetryItem.h>
+#include <mavros_msgs/ESCTelemetry.h>
+
+namespace mavros
+{
+namespace extra_plugins
+{
+/**
+ * @brief ESC status plugin
+ */
+class ESCTelemetryPlugin : public plugin::PluginBase
+{
+public:
+    ESCTelemetryPlugin() : PluginBase(),
+        nh("~")
+    {}
+
+    void initialize(UAS &uas_) override
+    {
```

Comment:
Please apply uncrustify as in contribution guide: https://github.com/mavlink/mavros/blob/master/CONTRIBUTING.md